.changeset/fuzzy-pumas-enjoy.md

```diff
@@ -0,0 +1,4 @@
+---
+'@finos/legend-application-studio': patch
+'@finos/legend-graph': patch
+---
```

Comment:
lets add a message here about adding option to generate OpenApi during service registration